deployment/ib-kubernetes.yaml

```diff
@@ -0,0 +1,132 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ib-kubernetes
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ib-kubernetes
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "patch", "watch"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["*"]
+    verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ib-kubernetes
+roleRef:
+  name: ib-kubernetes
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: ib-kubernetes
+    namespace: kube-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ib-kubernetes-config
+  namespace: kube-system
+data:
+  PLUGIN: "noop"
+  PERIODIC_UPDATE: "5"
+  GUID_RANGE_START: "02:00:00:00:00:00:00:00"
+  GUID_RANGE_END: "02:FF:FF:FF:FF:FF:FF:FF"
+  UFM_USERNAME: ""
```

Comment:
The UFM info should be in a secret and link like this see https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
added